README.md

```diff
@@ -87,15 +87,27 @@ oc create -f istio/ -n istio-system
 
 ## Routing service traffic through the adapter
 
+The following statements assume that configuration has been generated using the tool provided. Custom modifications
+to the match condition in the rule are possibly but explanation of any or all combinations is outside of the scope of
```

Comment:
s/possibly/possible/

I'd just say that other combinations should still be made taking into account the rule should still match and provide the needed information (ie. url, token and service_id).